src/Cpanel/PluginActions.php

```diff
@@ -13,6 +13,36 @@ class PluginActions extends AbstractPluginActions
     protected $dataStore;
     protected $logger;
     protected $request;
+    protected $userConfig;
+    protected $composer;
+
+    public static $CONFIG = array(
+        'debug' => false,
+        'featureManagerIsFullZoneProvisioningEnabled' => true,
+        'isDNSPageEnabled' => true,
+        'useHostAPILogin' => true,
+        'homePageCards' => array(
+            'AlwaysOnlineCard',
+            'IPV6Card',
+            'CacheLevelCard',
+            'RailgunCard',
+            'PurgeCacheCard',
+        ),
+        'moreSettingsCards' => array(
+            'container.moresettings.security' => array(
+                'SecurityLevelCard',
+                'ChallengePassageCard',
+                'BrowserIntegrityCheckCard',
+            ),
+            'container.moresettings.speed' => array(
+                'MinifyCard',
+                'DevelopmentModeCard',
+                'BrowserCacheTTLCard',
+            ),
+        ),
+        'locale' => 'en',
+        'integrationName' => 'cpanel',
+    );
```

Comment:
We don't want all of the fields to be configurable. I think the below fields should not be configurable by the `config.json` file.  

```
'isDNSPageEnabled' => true,
'useHostAPILogin' => true,
'integrationName' => 'cpanel',
```
@jwineman I just wrote the settings I think which shouldn't be configurable. I only have strong opinions on `integrationName` and `useHostAPILogin`. `isDNSPageEnabled` could be configurable...